src/platform/packages/shared/kbn-test/src/auth/session_manager.ts

```diff
@@ -211,4 +211,8 @@ Set env variable 'TEST_CLOUD=1' to run FTR against your Cloud deployment`
     const profileData = await getSecurityProfile({ kbnHost: this.kbnHost, cookie, log: this.log });
     return profileData;
   }
+
+  getSupportedRoles() {
+    return this.supportedRoles!.roles;
```

Comment:
Would this be a bit safer? Given the `supportedRoles!`, I wonder why we're so sure that `supportedRoles` has a defined value.

```suggestion
    return this.supportedRoles ? this.supportedRoles.roles : [];
```

A test like the following would fail

```ts
    test(`getSupportedRoles when no supported roles are supplied`, async () => {
      const samlSessionManager = new SamlSessionManager({
        ...samlSessionManagerOptions,
      });
      const roles = samlSessionManager.getSupportedRoles();
      expect(roles).toEqual([]);
    });
```

output:

````
    TypeError: Cannot read properties of undefined (reading 'roles')

      214 |
      215 |   getSupportedRoles() {
    > 216 |     return this.supportedRoles!.roles;
          |                                 ^
      217 |   }
      218 | }
      219 |
```

Reply:
done in f0c075d6730961ab1c78121e8e5141a4d0428965 good catch!